### ISSUE TITLE
Batching the Analysis of Multiple Games

### ISSUE THREAD

**Issue by psygo** [sanderland/katrain#722]
I typically play games in sessions, and analyze the games n batches of 3-5. I try to leave my KaTrain analyzing the games while I play, but sometimes it's doesn't go that smoothly.

I wish we had a feature for batch analysis, such that we could queue multiple game files and cache them in the SGF at the same time. That would be quite handy for when coming back from tournaments or in-person sessions.

**Comment by adzweig** [comment]
yes, this would be fantastic. It would also allow to run the analysis on a cloud computer and review it later locally.
Are there any plans to add this feature?

**Comment by psygo** [comment]
Apparently, [Lizzieyzy](https://github.com/yzyray/lizzieyzy) already has this feature.
